src/types.ts

```diff
@@ -61,9 +61,16 @@ export interface DirectusProviderProps {
   apiUrl: string;
   /** A set of options to pass to the Directus client. */
   options?: DirectusOptions;
+  /**
+   * If `true`, the provider will try to login the user automatically on mount.
+   * @default false
+   */
+  autoLogin?: boolean;
   children: React.ReactNode;
 }
 
+export type AuthStates = 'loading' | 'authenticated' | 'unauthenticated';
```

Comment:
Why not using an enum here?

Reply:
I don't realy have a firm stance in the Enum vs Union debate. If you prefere Enums i can update the code acordingly.
Should i update the DirectusImage fit and format props too?

Reply:
No problem then, let's go for unions.

Reply:
Okay.
I read a little more into it last night. Enums would sound kinda nice, but if we would use them, we should use them everywhere. Sadly that would be a breaking change.

Maybe for the Directus SDK v11 compatible Version?